app/coffee/ProjectManager.coffee

```diff
@@ -72,6 +72,22 @@ module.exports = ProjectManager =
 					else
 						callback(null)
 
+	queueFlushAndDeleteProject: (project_id, callback = (error) ->) ->
+		RedisManager.queueFlushAndDeleteProject project_id, (error) ->
+			if error?
+				logger.error {project_id: project_id, error:error}, "error adding project to flush and delete queue"
+				return callback(error)
+			Metrics.inc "queued-delete"
+			callback()
+
+	getProjectDocsTimestamps: (project_id, callback = (error) ->) ->
+		RedisManager.getDocIdsInProject project_id, (error, doc_ids) ->
+			return callback(error) if error?
+			return callback() if !doc_ids?.length
```

Comment:
I guess this could be `callback(null, [])` so the caller would always get an array back instead of having to check for absence?

Reply:
As above, thanks.